api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -31,11 +31,12 @@ message QuicProtocolOptions {
 
   // Maximum number of milliseconds that connection will be alive when there is
   // no network activity. 300000ms if not specified.
-  google.protobuf.Duration idle_timeout = 2;
+  google.protobuf.Duration idle_timeout = 2 [(validate.rules).duration = {gte {nanos: 1000000}}];
```

Comment:
These are breaking changes of a non-WIP API, which we typically do not allow.
That said, I think these changes make sense. @htuch WDYT?

Reply:
QUIC is still in Alpha. Do we allow API change before GA?

Reply:
If it's in alpha then I suggest marking the protos as WiP. We can't make changes generally to protos such as stronger PGV guards unless there is a `work_in_progress` annotation. I don't think we can do that retrospectively though if folks are already depending on this.

Reply:
If so, can we modify their comment and add the enforcement in the code?

Reply:
Modified the code instead. PTAL

Reply:
Yep, code is good, thanks.